src/test/java/org/folio/marc4ld/mapper/Marc2BibframeMapperIT.java

```diff
@@ -270,6 +270,7 @@ void map_shouldReturnCorrectlyMappedResource() {
     validateInstance(result);
     assertThat(result.getOutgoingEdges()).isNotEmpty();
     var edgeIterator = result.getOutgoingEdges().iterator();
+    validateWork(edgeIterator.next());
```

Comment:
I am actually little surprised that these tests are not intermittently failing. As `result.getOutgoingEdges()` returns a Set, there is no guarantee in the order in which edges are returned.